services/inventory/services.go

```diff
@@ -41,13 +41,24 @@ func NewServicesService(db *reform.DB, r registry) *ServicesService {
 	}
 }
 
+// ServiceFilters represents filters for agents list.
+type ServiceFilters struct {
+	// Return only Services that provide insights for that Node.
```

Comment:
"insights" is a term for Prometheus metrics and QAN data. In other word, that's for Agents, not Services and Nodes.